packages/manager/apps/pci-kubernetes/src/pages/new/steps/ClusterBillingStep.component.tsx

```diff
@@ -19,41 +20,54 @@ export function ClusterBillingStep({
   onSubmit,
   step,
 }: Readonly<BillingStepProps>) {
-  const { t: tStepper } = useTranslation('stepper');
+  const { t } = useTranslation('stepper');
   const [antiAffinity, setIsAntiaffinity] = useState(false);
   const [isMonthlyBilled, setIsMonthlyBilled] = useState(false);
 
   const isPricingComingSoon = form.flavor?.blobs?.tags?.includes(
     TAGS_BLOB.COMING_SOON,
   );
 
+  const getPrice = useMemo(() => {
```

Comment:
```suggestion
  const price = useMemo(() => {
```
this is a variable not a function